client/notification.js

```diff
@@ -97,6 +97,17 @@ function buildQueryStringFromDict(dictionary) {
   return queryString ? '?' + queryString : '';
 }
 
+function checkOptionsKeys(allowedKeys, options) {
+  var invalidKeys = Object.keys(options).filter((key_name) =>
```

Comment:
Any reason to be using `var` instead of `let` here? I noticed you’d changed a bunch of `var`s and globals to be `let` in a different commit.